pages/pipelines/defining_steps.md

```diff
@@ -124,6 +123,31 @@ When you run a pipeline, a build is created. Each of the steps in the pipeline e
 
 <%= image "job-states.svg", size: "1482x690", alt: "Job state diagram" %>
 
+Job State             | Description
+----------------------| -----------------------------------------
+PENDING               | The job has just been created and doesn't have a state yet
```

Comment:
Style:

- In the diagram above and text below it looks like these are normally written in lower case and using code styling. I think this should be consistent with that unless there's a good reason not to.
- End each description with a period.

```suggestion
`pending`               | The job has just been created and doesn't have a state yet.
```

^ This applies to each row.